### ISSUE TITLE
Time series: Can't zoom after edit panel

### ISSUE THREAD

**Issue by bobemoe** [grafana/grafana#71976]
### What happened?

Drag mouse over a sub section of a time series panel and it highlights correctly, but on mouse button release the time range does not update (no zoom happens)

### What did you expect to happen?

Time rage updates to the selected area creating zoom in function

### Did this work before?

Pretty sure this was not an issue in 9.x
Recently upgraded to 10-preview1 and now to 10.0.2

### How do we reproduce it?

1. Create new dashboard.
2. Add a visualisation: Grafana>Random walk.
4. Save the panel.
5. See time range is last 6 hours.
6. Drag over the middle of the panel to highlight and zoom to a specific time range.
7. Confirm time range updates.
9. Change time range back to last 6 hours.
10. Edit the panel. You don't need to make any changes, click either discard or save. 
11. Drag over the middle of the panel to highlight and zoom to a specific time range.
12. See the time range does not change this time.

### Environment (with versions)?

Grafana: 10.0.2
OS: Linux
Browser: Firefox

### Grafana platform?

Docker

**Comment by leeoniya** [comment]
any console errors in browser's DevTools?

**Comment by bobemoe** [comment]
Yep repeatable there too:

1. Load fresh https://play.grafana.org/
2. On "server requests" panel click "edit". Immediately click "discard".
3. Drag over the middle of "server requests" to highlight and zoom to a specific time range.
4. See the time range does not change.

Note: drag-zoom during edit mode always works, its only an issue once leaving edit mode.

If you repeat the above without step 2 then the issue is not triggered.

I've tried in Chromium now as well as FireFox. Same issue.

DevTools Console is pretty noisy but I cant see any obvious additional errors while edit/discard the panel or trying and failing to zoom.

I did however notice that panels that were not initially rendered (were off screen and not yet scrolled to) at the time of triggering the issue were not affected and allow a drag-zoom, while all panels that were initially rendered at time of issue trigger (even if off screen at the time) showed the issue.

Also reproduced on "State Timeline" so I don't think its limited to "Time Series"
